File: MIS440 Final Projects/Gcd & Prime & Phi/WindowsApp1/Form1.vb
Public Class Form1

    '-------------GCD---------------------
    Private Sub cmdgcd_Click(sender As Object, e As EventArgs) Handles cmdgcd.Click
        Dim a, b As Long
        If txtGcd1.Text <> "" And txtGcd2.Text <> "" Then
            a = txtGcd1.Text
            b = txtGcd2.Text
            txtGcdR.Text = gcdfun(a, b)
        End If
    End Sub

    '--------PHI------------------------
    Private Sub cmdphi_Click(sender As Object, e As EventArgs) Handles cmdphi.Click
        Dim a As Long
        If txtPin.Text <> "" Then
            a = txtPin.Text
            txtPhiR.Text = phifun(a)
        End If
    End Sub

    '-----------PRIME-------------------
    Private Sub cmdprime_Click(sender As Object, e As EventArgs) Handles cmdprime.Click
        Dim a As Long
        If txtPin.Text <> "" Then
            a = txtPin.Text
            If isprime(a) = True Then
                txtPrimeR.Text = "it is prime"
            Else
                txtPrimeR.Text = "it is not prime"
            End If
        End If
    End Sub

    '-------------OTHER------------------
    Private Sub Reset_Click(sender As Object, e As EventArgs) Handles Reset.Click
        txtGcd1.Clear()
        txtGcd2.Clear()
        txtGcdR.Clear()
        txtPin.Clear()
        txtPhiR.Clear()
        txtPrimeR.Clear()
    End Sub

    Private Sub txtGcd1_KeyPress(sender As Object, e As KeyPressEventArgs) Handles txtGcd1.KeyPress
        txtGcdR.Clear()
    End Sub
    Private Sub txtGcd2_KeyPress(sender As Object, e As KeyPressEventArgs) Handles txtGcd2.KeyPress
        txtGcdR.Clear()
    End Sub
    Private Sub txtPin_KeyPress(sender As Object, e As KeyPressEventArgs) Handles txtPin.KeyPress
        txtPhiR.Clear()
        txtPrimeR.Clear()
    End Sub

End Class

File: Presentation/GCD_C#/CSharpPre/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace CSharpPre
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        public void cmdgcd_Click(object sender, EventArgs e)
        {
            int a, b;
            if (txtGcd1.Text != "" & txtGcd2.Text != "") {
                a = Convert.ToInt32(txtGcd1.Text);
                b = Convert.ToInt32(txtGcd2.Text);
                txtGcdR.Text = Convert.ToString(gcdfun(a, b));
            }
        }

        //---------GCD------------------------------------------
        public int gcdfun(int a,int b)
        {
            int temp;
        while (b > 0){
                temp = b;
            b = a % b;
            a = temp;
        }
            return a;
    }
        private void Reset_Click(object sender, EventArgs e)
        {
            txtGcd1.Clear();
            txtGcd2.Clear();
            txtGcdR.Clear();

        }

    }
}
